index.js

```diff
@@ -69,6 +62,19 @@ db.once('open', async () => {
   } catch (error) {
     console.error(error)
   }
+}
+
+mongoose.connect(process.env.DB_URI, { useNewUrlParser: true, useUnifiedTopology: true });
+
+const db = mongoose.connection;
+
+db.on('error', (err) => {
+  console.error(err);
+});
+
+db.once('open', () => {
+  new CronJob('0 0 12 * * * *', () => putAllMoviePagesInDatabase(), null, true, 'America/Los_Angeles');
+  putAllMoviePagesInDatabase();
 });
 
 const server = http.createServer((req, res) => {
```

Comment:
I think we can remove this redundant callback now. Just start server without callback.